.github/workflows/ci.yml

```diff
@@ -431,6 +420,17 @@ jobs:
       project-name: Component[OpenTelemetry.Resources.Azure]
       code-cov-name: Resources.Azure
 
+  build-test-resources-container:
+    needs: detect-changes
+    if: |
+      contains(needs.detect-changes.outputs.changes, 'resources-container')
+      || contains(needs.detect-changes.outputs.changes, 'build')
+      || contains(needs.detect-changes.outputs.changes, 'shared')
+    uses: ./.github/workflows/Component.BuildTest.yml
+    with:
+      project-name: Component[OpenTelemetry.Resources.Container]
+      code-cov-name: ResourceDetectors.Container
```

Comment:
Should it be renamed to Resources.Container?

Reply:
Good catch. Fixed in e87aa684ab11485c7ea9349b22e20a29537ba466 with couple other things